### PR TITLE
Update Baidu.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by J0WI** [comment]
You removed `open2` and `yum` without a comment.

File: src/chrome/content/rules/Baidu.xml [hunk 1]
```diff
@@ -169,7 +166,10 @@
 	<target host="wallet.baidu.com" />
 	<target host="wap.baidu.com" />
 	<target host="www.baidu.com" />
-	<!--target host="yun.baidu.com" /-->
+	
+	<!--	https://github.com/EFForg/https-everywhere/pull/4584
```
Comment (J0WI) [inline src/chrome/content/rules/Baidu.xml]: Please just use `incomplete certificate chain`

File: src/chrome/content/rules/Baidu.xml [hunk 2]
```diff
@@ -152,13 +149,13 @@
 	<target host="8.baidu.com" />
 	<target host="baifubao.baidu.com" />
 	<target host="bs.baidu.com" />
-	<target host="cpro.baidu.com" />
+	<!--target host="cpro.baidu.com" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/Baidu.xml]: Please document why you did this

Comment (renyouguo) [inline src/chrome/content/rules/Baidu.xml]: Check error that time.
Ok now.

**Pull request merged**